### PR TITLE
remove redundant registryKeyPath. might be a bug?

### PR DESCRIPTION
**Pull request by SimonCropp**
## Bug

Fixes:

Regression? Last working version:

## Description
so registryKeyPath is not used. I dont know if this is a bug?

## PR Checklist

- [x] PR has a meaningful title
- [ ] PR has a linked issue.
- [x] Described changes

- **Tests**
  - [ ] Automated tests added
  - **OR**
  
  - [ ] Test exception
  - **OR**
  - [x] N/A 

- **Documentation**
  
  - [ ] Documentation PR or issue filled
  - **OR**
  - [x] N/A

### CODE REVIEW

**Comment by erdembayar** [comment]
@SimonCropp 
Thank you for your contribution.  Could you be able create tracking issue at https://github.com/nuget/home/issues/new and link at next to `Fixes` in description?

**Comment by SimonCropp** [comment]
> Could you be able create tracking issue at https://github.com/nuget/home/issues/new and link at next to Fixes in description?

in this context, what value does having a tracking issue add. Seems to be extra effort for no benefit

**Comment by SimonCropp** [comment]
i mean a PR is just an issue with code attached.

**Comment by erdembayar** [comment]
@dominoFire 
FYI: It looks this PR overlaps with your https://github.com/nuget/client.engineering/issues/640 effort.

File: src/NuGet.Clients/NuGet.CommandLine/RegistryKeyUtility.cs [hunk 1]
```diff
@@ -9,7 +9,7 @@ namespace NuGet.CommandLine
 {
     public static class RegistryKeyUtility
     {
-        public static object GetValueFromRegistryKey(string name, string registryKeyPath, RegistryKey registryKey, ILogger logger)
+        public static object GetValueFromRegistryKey(string name, RegistryKey registryKey, ILogger logger)
```
Comment (dominoFire) [inline src/NuGet.Clients/NuGet.CommandLine/RegistryKeyUtility.cs:12]: Actually, `registryKeyPath` parameter is the registry key path open in `RegistryKey` object. Name is the 'name of the vaulue.

https://github.com/NuGet/NuGet.Client/blob/cc1e4ca108a97dc2af269ad5b80e94314c6b9f6c/src/NuGet.Clients/NuGet.CommandLine/RegistryKeyUtility.cs

We are more than happy to accept this contribution if you can change `name` to `registryKeyPath` in line 16 and add some unit tests.

Comment (SimonCropp) [reply src/NuGet.Clients/NuGet.CommandLine/RegistryKeyUtility.cs:12]: how would i add a unit test for an un-used parameter?

Comment (nkolev92) [reply src/NuGet.Clients/NuGet.CommandLine/RegistryKeyUtility.cs:12]: @dominoFire 

Can that be handled separately? 
This PR focuses on removing the unused param only. 

@SimonCropp 

The build is failing with the following: 

```
##[error]src\NuGet.Clients\NuGet.CommandLine\Program.cs(31,30): Error CA1823: Unused field 'FilesystemRegistryKey'.
     8>C:\a\1\s\src\NuGet.Clients\NuGet.CommandLine\Program.cs(31,30): error CA1823: Unused field 'FilesystemRegistryKey'. [C:\a\1\s\src\NuGet.Clients\NuGet.CommandLine\NuGet.CommandLine.csproj]
##[error]src\NuGet.Clients\NuGet.CommandLine\Program.cs(30,30): Error CA1823: Unused field 'OSVersionRegistryKey'.
     8>C:\a\1\s\src\NuGet.Clients\NuGet.CommandLine\Program.cs(30,30): error CA1823: Unused field 'OSVersionRegistryKey'. [C:\a\1\s\src\NuGet.Clients\NuGet.CommandLine\NuGet.CommandLine.csproj]
##[error]src\NuGet.Clients\NuGet.CommandLine\Program.cs(32,30): Error CA1823: Unused field 'DotNetSetupRegistryKey'.
```

Can you please take a look at that?

Comment (dominoFire) [reply src/NuGet.Clients/NuGet.CommandLine/RegistryKeyUtility.cs:12]: I created a new PR to use `registryKeyPath` parameter. More details at: https://github.com/NuGet/NuGet.Client/pull/4346

**Comment by dominoFire** [comment]
Thanks for your contribution, @SimonCropp ; unfortunately, we cannot take this PR since the parameters are now used in `RegistryKeyUtility` class. See link below:

https://github.com/NuGet/NuGet.Client/blob/914cacea77660b8758014912ee127c82469ac1d9/src/NuGet.Clients/NuGet.CommandLine/RegistryKeyUtility.cs#L16

We really appreciate your time and your effort to contribute to NuGet. 

You are more than welcome to contribute with issues, PRs and discussions! Any thoughts, please let us know!

**Pull request closed without merging**